apidoc/Titanium/UI/UI.yml

```diff
@@ -1507,9 +1507,31 @@ properties:
         `KEYBOARD_DEFAULT`.
     type: Number
     permission: read-only
-    
+
+  - name: KEYBOARD_WEBSEARCH
+    summary: Use a keyboard optimized for web search terms and URL entry.
+    description: |
+        Use with the <Titanium.UI.TextField.keyboardType> and <Titanium.UI.TextArea.keyboardType>
+        properties. This type features the space and “.” characters prominently.
```

Comment:
running node validate in apidocs found error in this line.
Don't use "." here. use `.` instead.